src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -1,6 +1,6 @@
 angular
 .module('impac.services.highcharts-factory', [])
-.factory('HighchartsFactory', ($filter)->
+.factory('HighchartsFactory', ($filter, $timeout)->
```

Comment:
This factory is starting to be monstrous unfortunately :/
Can you leverage your FE discussions with @alexnoox to find a smart way of grouping the Highcharts formatting without compromising the readability / code maintainability?